src/xdocs/config_imports.xml

```diff
@@ -1433,11 +1433,21 @@ import java.util.stream.IntStream;
             <td>true</td>
             <td>3.3</td>
           </tr>
+          <tr>
+            <td>staticGroups</td>
+            <td>
+                list of <b>static</b> import groups (every group identified either by a
+                common prefix string, or by a regular expression enclosed
+                in forward slashes (e.g. <code>/regexp/</code>)
```

Comment:
please define here what does empty value ("") mean for new property.
It is better to do the same for `groups` property too.

Reply:
done